test/script_tools.py

```diff
@@ -36,6 +36,7 @@
 from vsc.install.testing import TestCase
 from vsc.utils.script_tools import ExtendedSimpleOption, DEFAULT_OPTIONS, CLI
 
+import vsc.utils.nagios
```

Comment:
why is this needed?

Reply:
It's not, it was a futile attempt to get mock to find the right code.